misc/object-conditions/basics.rst

```diff
@@ -1,443 +1,438 @@
+  | Matches if one specific given field value is selected.
+* | **contains all not**:
+  | Matches if all given field values are not selected.
+* | **contains one not**:
+  | Matches if one specific given field value is not selected.
 
 
-Text field (input)
-++++++++++++++++++
+Single Tree Selection Field
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Check if any field of type text contains a specific string (or not)?
-The configured "Type" of input fields has no impact on the available options.
+Checks if a selected field value match the attribute value.
+Selecting more than one value in the condition will act like an "OR" clause.
 
 Available operators for matching:
 
-   *contains*
-      Matches if text contains a specific string.
-
-   *contains not*
-      Matches if text does not contain a specific string.
-
-   *is any of*
-      Matches if text is equal to any of given tokens.
-
-   *is none of*
-      Matches if text is not equal to all of given tokens.
+- **is**
+- **is not**
+- **has changed**
 
-   *starts with one of*
-      Matches if text starts with one of given tokens.
+Multiple Tree Selection Field
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-   *ends with one of*
-      Matches if text ends with one of given tokens.
+Checks if selected field value(s) match the attribute value(s).
 
-   *matches regex*
-      Evaluates if text matches provided regular expression.
-
-   *does not match regex*
-      Evaluates if text does not match provided regular expression.
+Available operators for matching:
 
-.. hint::
-   **Regex support**
+* | **contains all**
+  | Matches if all given field values are selected.
+* | **contains one**
+  | Matches if one specific given field value is selected.
+* | **contains all not**
+  | Matches if all given field values are not selected.
+* | **contains one not**
+  | Matches if one specific given field value is not selected.
 
-   *matches regex* and *does not match regex* are supported only in
-   Triggers, Time Accounting selector, Postmaster Filters, Automatic ticket
-   assignment and Core Workflow.
+Additional Information and Remarks
+----------------------------------
 
-.. hint::
-   **What about my "old" style** ``regex:...`` **filters?**
+General
+^^^^^^^
 
-   If you update your Zammad from 6.0 or prior and you have already conditions
-   with *contains* or *contains not* including a regex filter
-   (i.e. ``regex:^(foo|bar)$``), Zammad tries to migrate them to the new
-   *matches regex* and *does not match regex* operators.
+Ticket context conditions do not support:
+   * regular expressions
+   * case sensitive string conditions
+   * Basic conditions do not support the same attribute to be selected more
+     than once
 
-.. hint::
-   **Differences in input fields**
+*has changed* conditions:
+   * Are only available for ticket attributes
+   * Don't affect ticket articles attributes
+   * Don't affect organization attributes
+   * Don't affect user attributes
+   * Don't affect group attributes
```

Comment:
You could consider merging the don't affect entries into one line (with a comma separated list)